SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-3761/SpatialTestCrossServerRPC/SpatialTestCrossServerRPC.cpp

```diff
@@ -33,43 +46,118 @@ void ASpatialTestCrossServerRPC::PrepareTest()
 {
 	Super::PrepareTest();
 
-	TArray<FVector> CubesLocations;
-	CubesLocations.Add(FVector(250.0f, 250.0f, 75.0f));
-	CubesLocations.Add(FVector(250.0f, -250.0f, 75.0f));
-	CubesLocations.Add(FVector(-250.0f, -250.0f, 75.0f));
-	CubesLocations.Add(FVector(-250.0f, 250.0f, 75.0f));
-
-	AddStep(TEXT("EnsureSpatialOS"), FWorkerDefinition::Server(1), nullptr, [this]() {
+	// Pre-test checks
+	AddStep(TEXT("Pre-test check"), FWorkerDefinition::Server(1), nullptr, [this]() {
 		USpatialNetDriver* SpatialNetDriver = Cast<USpatialNetDriver>(GetNetDriver());
+		SpatialNetDriver = Cast<USpatialNetDriver>(GetNetDriver());
 
 		if (SpatialNetDriver == nullptr || SpatialNetDriver->LoadBalanceStrategy == nullptr)
 		{
 			FinishTest(EFunctionalTestResult::Error, TEXT("Test requires SpatialOS enabled with Load-Balancing Strategy"));
 		}
 		else
 		{
-			FinishStep();
+			CheckInvalidEntityID(LevelCube);
```

Comment:
Why is this here?

Reply:
This check was useful before the fix but it is redundant now, so I have removed it.